rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -17,6 +17,31 @@
         <tag>reviewed-2015-06-17</tag>
     </metadata>
     <rules>
+        <rule id="websphere-catchall-db2-00000">
+            <when>
+                <javaclass references="com.ibm.db2.jcc.{type}" />
+            </when>
+            <perform>
+                <iteration>
+                    <when>
+                        <not>
+                            <has-hint />
+                        </not>
+                    </when>
+                    <perform>
+                        <hint title="IBM DB2 driver type reference" effort="1" severity="potential">
```

Comment:
Yeah, my main concern here is making sure that we don't attach effort to rules that will often trigger thousands of times, because it causes performance issues and useless story points numbers.

I'm fine with checking it in with effort first, and then looking at the perfcheck and test apps a few days later to see what the impact is. (I think we need to do this for any large rules PR.)

I do think if a rule needs effort it should be in a different ruleset from the catchalls, but I don't feel strongly about that part.